gatorgrouper/tests/test_group_method.py

```diff
@@ -60,6 +71,25 @@ def test_group_random_extra():
     assert num_group == 2
 
 
+@given(grpsize=integers(min_value=1, max_value=3))
+@settings(verbosity=Verbosity.verbose)
+@pytest.mark.hypothesisworks
+def test_group_random2(grpsize):
+    """This hypothesis test will test the test_group_random_extra"""
```

Comment:
This docstring is incorrect -- this test only tests the `group_random_group_size` method. It is, however, a good model for a hypothesis test.

Reply:
I changed the docstring.